sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubServiceClient.cs

```diff
@@ -216,6 +216,8 @@ public virtual Response CloseAllConnections(IEnumerable<string> excluded = null,
         public virtual async Task<Response> SendToAllAsync(RequestContent content, ContentType contentType, IEnumerable<string> excluded = null, RequestContext context = null)
 #pragma warning restore AZC0002
         {
+            Argument.AssertNotNull(content, nameof(content));
```

Comment:
It is a breaking change to introduce an exception where there wasn't one before, and this library has GA'ed

Reply:
@AlexanderSher ^^